<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageROIEmbeddedWidget</class>
 <widget class="QWidget" name="ImageROIEmbeddedWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>206</width>
    <height>83</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="mpApplyButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
  <widget class="QPushButton" name="mpResetButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="mpUseCrossHairCheckbok">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>91</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Use CrossHair</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="mpFromCenterCheckbox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>91</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>From Center</string>
   </property>
  </widget>
  <widget class="QPushButton" name="mpBuiltInSelectorButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Built-in Selector</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
